### PR TITLE
[SW-2016] Expose getDomainValues on H2OMojoModel

### PR DESCRIPTION
**Pull request by jakubhava**

### CODE REVIEW

File: scoring/src/main/scala/ai/h2o/sparkling/ml/models/H2OMOJOModel.scala [hunk 1]
```diff
@@ -46,6 +46,21 @@ class H2OMOJOModel(override val uid: String) extends H2OMOJOModelBase[H2OMOJOMod
 
   def getModelDetails(): String = $(modelDetails)
 
+  def getDomainValues(): Array[Array[String]] = {
```
Comment (mn-mikke) [inline scoring/src/main/scala/ai/h2o/sparkling/ml/models/H2OMOJOModel.scala]: Shouldn't we return `Map[String, Array[String]]` where key is a column name instead?

File: ml/src/test/scala/ai/h2o/sparkling/ml/algos/MultinomialPredictionTestSuite.scala [hunk 2]
```diff
@@ -113,4 +113,15 @@ class MultinomialPredictionTestSuite extends FunSuite with Matchers with SharedH
     assert(schema == expectedSchema)
     assert(schema == expectedSchemaByTransform)
   }
+
+  test("getDomainValues") {
```
Comment (mn-mikke) [inline ml/src/test/scala/ai/h2o/sparkling/ml/algos/MultinomialPredictionTestSuite.scala]: In fact, this test is testing `getDomainValuesForCol` method. Shouldn't we change name of this test and introduce another one for  `getDomainValues`?

File: scoring/src/main/scala/ai/h2o/sparkling/ml/models/H2OMOJOModel.scala (hunk 1 above)
Comment (mn-mikke) [inline scoring/src/main/scala/ai/h2o/sparkling/ml/models/H2OMOJOModel.scala]: Do Python and R converters support nested collections? Shouldn't we cover it with tests?

File: ml/src/test/scala/ai/h2o/sparkling/ml/algos/MultinomialPredictionTestSuite.scala (hunk 2 above)
Comment (jakubhava) [reply ml/src/test/scala/ai/h2o/sparkling/ml/algos/MultinomialPredictionTestSuite.scala]: 👍  Good catch, I should introduce more tests anyway

File: scoring/src/main/scala/ai/h2o/sparkling/ml/models/H2OMOJOModel.scala (hunk 1 above)
Comment (jakubhava) [reply scoring/src/main/scala/ai/h2o/sparkling/ml/models/H2OMOJOModel.scala]: Yup, better idea!

Comment (jakubhava) [reply scoring/src/main/scala/ai/h2o/sparkling/ml/models/H2OMOJOModel.scala]: Good idea, will test

Comment (jakubhava) [reply scoring/src/main/scala/ai/h2o/sparkling/ml/models/H2OMOJOModel.scala]: With this I think we should keep only one method `def getDomainValues(): Map[String, Array[String]]`, users can extract the specific column themselves

Comment (mn-mikke) [reply scoring/src/main/scala/ai/h2o/sparkling/ml/models/H2OMOJOModel.scala]: Yup, makes sense!

Comment (jakubhava) [reply scoring/src/main/scala/ai/h2o/sparkling/ml/models/H2OMOJOModel.scala]: Fixed Python & added test for Python & R

**Pull request merged**